src-cljs/cuttle/util.cljs

```diff
@@ -19,6 +20,16 @@
 ;;------------------------------------------------------------------------------
 ;; Util Functions
 ;;------------------------------------------------------------------------------
+(defn not-empty?
```

Comment:
I'm not crazy about this function. Feels like it should be equivalent to `(not (empty? my-coll))` and just used in that way directly in the code.